MSAL/test/unit/MSALDeviceInfoProviderTests.m

```diff
@@ -385,6 +388,12 @@ - (void)testGetDeviceInfo_whenSSOExtensionPresent_andReturnedDeviceInfo_shouldRe
         XCTAssertEqualObjects(deviceInformation.extraDeviceInformation[@"isSSOExtensionInFullMode"], @"No");
         XCTAssertEqualObjects(deviceInformation.extraDeviceInformation[MSID_BROKER_MDM_ID_KEY], @"mdmId");
         XCTAssertEqualObjects(deviceInformation.extraDeviceInformation[MSID_ENROLLED_USER_OBJECT_ID_KEY], @"objectId");
+#if TARGET_OS_OSX && __MAC_OS_X_VERSION_MAX_ALLOWED >= 130000
+        XCTAssertEqualObjects(deviceInformation.extraDeviceInformation[MSID_PLATFORM_SSO_STATUS], @"MSIDPlatformSSOEnabledAndRegistered");
```

Comment:
MSAL developers won't be able to access an MSID-namespace enum. 
I don't think it makes sense for it to be a string either, should it be an MSAL-namespace enum instead?

Reply:
I will change it to MSAL enum , but ExtraDeviceInfo Dict Key values are expected to be string, so shall i keep the value as string ?

Reply:
Or do we want to return platform sso status outside of extraDeviceInfo ? like device mode ?

Reply:
Yeah, we can keep it as string too, just consistent format with other keys then. I was confused, because you were using enum, but returning it as string

Reply:
Updated logic to return platform sso status on mac os as an enum property on DeviceInfo